src/polyswarm/client/report.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import absolute_import
+import logging
+import os
+
+import click
+
+
+from polyswarm.client import utils
+
+logger = logging.getLogger(__name__)
+
+
+@click.group(short_help='Interact with the Polyswarm reporting system.')
+def report():
+    pass
+
+
+SECTIONS = "summary, detections, fileMetadata, network, droppedFiles, extractedConfig, analysis"
+
+
+@report.command('create', short_help='Create a report for an instance or sandbox id.')
+@click.argument('format', type=click.Choice(['html', 'pdf']))
+@click.argument('type', type=click.Choice(['scan', 'sandbox']))
+@click.argument('object-id', callback=utils.validate_id)
+@click.option('--template-id', metavar='ID', type=click.STRING)
+@click.option('--includes',
+              help=f'Comma-separated list of sections to include in the report. Can be one ore more of: {SECTIONS}',
+              multiple=True,
+              callback=lambda _,o,x: x[0].split(',') if len(x) == 1 else x)
```

Comment:
When you provide `multiple=True` you don't need the callback to split things in `click`, it automatically builds a list when you provide the `--includes` multiple times, e.g.:

```
--includes header --includes body --includes footer
```

ends up in the `includes` variable as a list `['header', 'body', 'footer']`. I would remove the callback since it is unnecessary code really.

Reply:
The reason is to also allow the user to provide it as `--includes header,body,footer` instead of `--includes header --includes body --includes footer` that is also valid. I copy and pasted this snipped from another command.